### PR TITLE
release 0.13

### PR DESCRIPTION
**Pull request by wenchajun**
Signed-off-by: chengdehao <wenchajun@gmail.com>

### CODE REVIEW

**Comment by benjaminhuo** [comment]
@wanjunlei @zhu733756 please help to review this

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+- [FEATURE] Add main workflow actions. #152
+- [FEATURE] Add support for rewrite tag. #155
+- [FEATURE] Add aws filter plugin. #173
+- [FEATURE] Add multiline filter plugin. #176
+- [FEATURE] Firehose plugin support. #178
+- [FEATURE] Add volume crd. #186
+- [ENHANCEMENT] Add priority class to FluentBit type. #146
+- [ENHANCEMENT] Upgrade layout from Kubebuilder v2 to Kubebuilder v3.1. #147
+- [BUGFIX] Fix some bugs.
+
```
Comment (zhu733756) [inline CHANGELOG.md:12]: This part can copy from v1.0.0 release docs. Ref: https://github.com/fluent/fluent-operator/blob/release-1.0/CHANGELOG.md#features

Comment (wenchajun) [reply CHANGELOG.md:16]: Thanks, I've amended it a bit. But this is a RELEASE of fluentbit-operator v0.13.0, not of fluentd-operator. So it doesn't add the part about fluentd-operator.

File: charts/fluentbit-operator/values.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 operator:
   image: "kubesphere/fluentbit-operator"
-  tag: "v0.12.0"
+  tag: "v0.13.0"
```
Comment (benjaminhuo) [inline charts/fluentbit-operator/values.yaml:6]: The content in VERSION file should also be changed to v0.13.0

**Comment by benjaminhuo** [comment]
You should checkout branch release-0.13 from branch `https://github.com/fluent/fluent-operator/tree/fluentbit-operator` , and then submit your PR to `release-0.13` of the main repo.  Branch `fluentbit-operator` is the master branch for fluentbit-operator @wenchajun

**Comment by wenchajun** [comment]
> 
OK, I get it. Since that branch doesn't have a corresponding release, I'll commit to this branch first and I'll commit the next pr to the `fluentbit-operator` branch later.

**Comment by benjaminhuo** [comment]
Branch `fluentbit-operator` is one commit ahead of `release-0.13.0`: https://github.com/fluent/fluent-operator/commit/9476bac51b430f9afd5fed25ec3749e5957b810d

And its version file should be latest, while the version file of `release-0.13.0` is 0.13.0

You cannot merge `release-0.13.0` back to `fluentbit-operator`

**Review by zhu733756 — APPROVED** [review]

**Comment by benjaminhuo** [comment]
@wenchajun not signed

x DCO — DCO

**Review by benjaminhuo — APPROVED** [review]

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**